Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -26,6 +26,16 @@ HardwareBreakpointManager::HardwareBreakpointManager(Target::Process *process)
 
 HardwareBreakpointManager::~HardwareBreakpointManager() {}
 
+ErrorCode HardwareBreakpointManager::add(Address const &address, Type type,
+                                         size_t size, Mode mode) {
+  int filled_sites = _sites.size();
```

Comment:
`filledSites`, or even `filled`, or even no temp variable (an additional "call" to `_sites.size()` is no problem in an assertion).